.env.example

```
@@ -0,0 +1 @@
+USER_EMAIL="simon.personale.email@example.com"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)